DEVELOPMENT.md

```diff
@@ -210,6 +210,13 @@ script again or you can manually update your conan config:
 conan config install contrib/conan/configs/[windows,linux]
 ```
 
+### How can I use aseparate debugging symbols for linux binaries?
+
+Orbit supports loading symbols from your workstation. Simply add directories that contain debugging symbols to the `SymbolPaths.txt` file. This file can be found at
+* Windows: `C:\Users\<user>\AppData\Roaming\OrbitProfiler\config\SymbolPaths.txt`
+* Linux: `~/orbitprofiler/config/SymbolPaths.txt`
+
+The symbols file must have a matching build id and the same name as the binary, but the extension `.debug` (e.g. `game.debug` or `game.elf.debug`).
```

Comment:
The build-id is checked right?  It's not the user's responsibility to ensure the build-id's match, I think we should make that clear in the comment.

Reply:
Yes, the build id is checked. I definitely see your point and reworded this a little bit. Is it better now?

Reply:
Yep, perfect, thanks!